gframe/game_config.cpp

```diff
@@ -160,6 +160,8 @@ bool GameConfig::Load(const path_char* filename)
 				draw_field_spell = !!std::stoi(str);
 			else if (type == "quick_animation")
 				quick_animation = !!std::stoi(str);
+			else if (type == "alternative_phase_layout")
+				alternative_phase_layout = !!std::stoi(str);
```

Comment:
Consider using the `DESERIALIZE_BOOL` macro. I know I haven't cleaned up this whole file yet but I do it for new additions.